DataSetInfoAbstractClass.py

```diff
@@ -0,0 +1,72 @@
+"""
+shared-latent-space/DataSetInfoAbstractClass.py
+
+This class defines an abstract class which is specific to a given
+type of data. Each data set should have its own implementation of
+this data set which defines both the functions load() and visualize().
+These functions are then used in shared_vae_class.py
+
+Author: Chris Williams
+Date: 5/22/18
+"""
+
+from abc import ABCMeta, abstractmethod
+
+
+class dataSetInfoAbstract(object):
+    __metaclass__ = ABCMeta
+
+    @abstractmethod
+    def load(self):
+        """
+        Loads the testing and training data from pickle files.
+        This needs to return the specified data.
+
+        Args: None
+
+        Returns: (Float array, Float array, Float array, Float array)
```

Comment:
Well I didn't see a way to define the return types for an abstract class. From what I read, it seems like Python's abstract classes aren't really a full abstract class because they don't do any type checking. This seemed like a way to inform the reader of what the function should return in all implementations.